sdk/helper/testcluster/docker/environment.go

```diff
@@ -555,7 +555,13 @@ func (n *DockerClusterNode) apiConfig() (*api.Config, error) {
 	if config.Error != nil {
 		return nil, config.Error
 	}
-	config.Address = fmt.Sprintf("https://%s", n.HostPort)
+
+	protocol := "https"
+	if transport.TLSClientConfig.Certificates == nil {
```

Comment:
I'm kinda surprised that `TLSClientConfig` is not nil leading to a panic above in the `n.TLSConfig()` call above? The DockerClusterNode only sets it's tlsConfig member variable from within the `setupCert`  function, so if we don't call that